Spigot-API-Patches/0218-Create-Player-sendActionBar-with-component-support.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jan Tuck <jan@tuck.dk>
+Date: Fri, 17 Jul 2020 01:54:22 +0200
+Subject: [PATCH] Create Player#sendActionBar with component support
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 2eb3121386e3bc5ccdd74726a7c4b5f832ec5aea..dfd04c341f6681e64cae21637395b59088b16d7a 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -646,6 +646,17 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+         return banEntry;
+     }
+ 
++    // Paper - Start
++    /**
++     * Sends an Action Bar message to the client.
++     *
++     * Use Section symbols for legacy color codes to send formatting.
++     *
++     * @param components The message to send in components
++     */
++    public void sendActionBar(@NotNull net.md_5.bungee.api.chat.BaseComponent... components);
```

Comment:
We don't need this, an API exist that works just fine for it.

Previously I marked deprecated on the component API because there was zero reason to use components there.

We can simply remove that deprecation in string based action bar api patch instead.

now, adding an API only patch for .sendActionBar like you have here that simply fowards to .sendMesage(ChatMessageType.ACTION_BAR, comps); would be ok as a utility method, but this should all just be merged into my original patch.